apps/e2e-tests/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "e2e-tests",
```

Comment:
are you sure tests should be one of the apps? how about moving it to root? technically e2e is not the app.

or maybe we can have
```
apps/
packages/
test/
--e2e

```

so we can keep apps folder clean but also we can scope e2e code inside nested package json?